src/Export/VTK.jl

```diff
@@ -56,9 +60,10 @@ end
 
 function WriteVTK.collection_add_timestep(pvd::WriteVTK.CollectionFile, datfile::VTKGridFile, time::Real)
     WriteVTK.collection_add_timestep(pvd, datfile.vtk, time)
+    return
```

Comment:
Since this extends WriteVTK, would it make sense to return the same as WriteVTK does instead of nothing?